src/main/java/jpa/entity/Skills.java

```diff
@@ -0,0 +1,88 @@
+package jpa.entity;
+
+import jpa.entity.Developers;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import lombok.extern.log4j.Log4j;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import javax.persistence.PostLoad;
+import javax.persistence.PostPersist;
+import javax.persistence.PostRemove;
+import javax.persistence.PostUpdate;
+import javax.persistence.PrePersist;
+import javax.persistence.PreRemove;
+import javax.persistence.PreUpdate;
+import javax.persistence.Table;
+import javax.persistence.MappedSuperclass;
+
+import java.util.HashSet;
+import java.util.Set;
+
+@MappedSuperclass
+@Log4j
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@ToString
+@Entity
+@Table(name = "skills")
+public class Skills {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column(name = "language", length = 255)
+    private String language;
+
+    @Column(name = "level", length = 255)
+    private String level;
```

Comment:
For junior we could use "j", "beginner", "jun" or something like this. 
But could we be more specific?
Maybe it is better to use not string?

Reply:
I think that better use full name) but it will better I can use char